### PR TITLE
add-vnet-service-endpoint

### PR DESCRIPTION
**Pull request by calvix**
add vnet service endpoint for storage

### CODE REVIEW

**Comment by calvix** [comment]
tested storage and everything works fine

File: service/controller/v9/resource/deployment/template/virtual_network_setup.json [hunk 1]
```diff
@@ -87,7 +87,12 @@
               },
               "routeTable":{
                 "id":"[parameters('routeTableID')]"
-              }
+              },
+              "serviceEndpoints": [
```
Comment (kopiczko) [inline service/controller/v9/resource/deployment/template/virtual_network_setup.json]: What does it do?

Comment (calvix) [reply service/controller/v9/resource/deployment/template/virtual_network_setup.json]: https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-service-endpoints-overview

**Comment by calvix** [comment]
this is bullshit again DNS error, just merging

**Pull request merged**